obspy/io/sac/sactrace.py

```diff
@@ -1158,10 +1176,23 @@ def read(cls, source, headonly=False, ascii=False, byteorder=None,
                                             checksize=checksize)
         if not debug_strings:
             for i, val in enumerate(hs):
-                val = _ut._clean_str(val, strip_whitespace=False)
+                if encoding.lower() != 'ascii':  # throw warning
+                    msg = ("Using encoding other than ASCII: {}\n"
+                           "All non-ASCII characters will be replaced by "
+                           "?".format(encoding))
```

Comment:
I just noticed that the `_clean_str()` method does already raise a warning in that case. So maybe just remove that warning here?